module2/homework/nwdNww/nwdNww.hpp

```diff
@@ -3,9 +3,44 @@
 int NWD(int lhs, int rhs) {
     // TODO: Implement me :)
     return -1;
+
+    if (rhs == 0){
+        return lhs;
+    }
+    if (lhs == 0){
+        return rhs;
+    }
+
+    if (lhs < 0){
+        lhs = -1*lhs;
+    }
+    if (rhs < 0){
+        rhs = -1*rhs;
+    }
```

Comment:
You can use an absolute value function (`std::abs()`)

Reply:
And one more thing here, you are opening brackets without space. And I see that you are using this space in other parts of code:
```suggestion
    if (lhs < 0) {
        lhs = -1 * lhs;
    }
    if (rhs < 0) {
        rhs = -1 * rhs;
    }
```
And if you don't want to use `std::abs()` please use whitespaces around operators.